Makefile

```diff
@@ -30,15 +30,15 @@ PYTHON_TARGET := 2.7
 REQUIREMENTS := test-requirements.txt requirements.txt
 PIP_OPTIONS := $(ST2_PIP_OPTIONS)
 
-NOSE_OPTS := --rednose --immediate
+NOSE_OPTS := --rednose --immediate --with-parallel
 NOSE_TIME := $(NOSE_TIME)
 
 ifdef NOSE_TIME
-	NOSE_OPTS := --rednose --immediate --with-timer
+	NOSE_OPTS := --rednose --immediate --with-parallel --with-timer
```

Comment:
Just FYI, adding `--with-parallel` is not enough, it will work only if `NODE_TOTAL` and `NODE_INDEX` env variables provided according to https://github.com/dlanger/nose-parallel

So the dev env remains the same as before.